### ISSUE TITLE
Validate default values

### ISSUE THREAD

**Issue by bnhovde** [alfrcr/predatorjs#3]
One thing that I've noticed is that I can't seem to get validation working when a form has default values. I need to manually go in and trigger the onChange for the validation to kick in (and the `formIsValid` value to become true)

Do you have any tricks for getting around this? I've tried running 'validate' for the fields on componentDidMount, but no luck.

**Comment by alfrcr** [comment]
Yes I have.
I have added a new method to trigger change all the form input. 
Here is the idea.
Add new method to dispatch all form with onChange event
```javascript
initializeForm = selector => {
        const form = document.querySelectorAll(`${selector} input:not(button):not(hidden):not(file)`)
        const event = new Event('input', { bubbles: true })
        _.each(form, f => setTimeout(() => f.dispatchEvent(event), 10))
      }
```

Then I pass initializeForm to WrappedComponent

```javascript
render() {
        return (
          <ComposedComponent
            {...this.props}
            initializeForm={this.initializeForm}
            validate={this.validate}
            getErrorMessage={this.getErrorMessage}
            getErrorMessages={this.getErrorMessages}
            formIsValid={this.state.formIsValid}
          />
        )
      }
```

Last thing to do is init the initializeForm() on your component in componentDidMount() lifecycle

```javascript

  componentDidMount() {
    this.props.initializeForm('#your-form-id')
  }

```

I will update these (with your PR) tonight. (8pm UTC +7)
Now I have a lot work to do at office.
Cheers. 😁

PS: btw do you have a good idea to naming the `initializeForm`? I think there's better name.

**Comment by bnhovde** [comment]
Great. That looks good! Nice work.

I think `initializeForm` works. Either that or `onInitialize`.

ps: Sorry for bombarding you 😸 , have a good day at the office.

**Comment by alfrcr** [comment]
Closed with ac4be307df6acba5e2e26122811add108370564d
